index.html

```diff
@@ -4,9 +4,65 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>First Portfoloio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet">
 </head>
 <body>
-    
+  <header class="header">
+    <div class="left">
+      <a href="#">
+        <div class="logo">
+            My logo
+        </div>
+    </a>
+    </div>
+    <nav class="right">
+        <div class="menubar">
+            <div class="menubar1"></div>
+            <div class="menubar2"></div>
+            <div class="menubar3"></div>
+        </div>
+    </nav>
+</header>
+<section class="container">
+  <div class="name">
+    <h1>I’m Sadaf.</h1>
+    <span>Glad to see you!</span>
+</div>
+<div class="bio">
+  <p>I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+</div>
+        <div class="lets_contact">
+          <div class="contact">
+            <h2>LET’S CONNECT</h2>
+        </div>
+            <section class="social">
+                <ul class="social-icons-set">
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-twitter" target="_blank" aria-label="Twitter"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-linkedin-in" target="_blank" aria-label="Linkedin"></i></a>
+                  </li>
+                  <li class="social-icon"><a href="#"><i class="fa-solid fa-m" target="_blank"
+                        aria-label="Medium"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-github" target="_blank" aria-label="GitHub"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-angellist" target="_blank" aria-label="Angellist"></i></a>
+                  </li>
+                </ul>
+              </section>
```

Comment:
- [ ] Kindly I suggest you use the `div` tag instead of the `section` tag here to follow HTML best practices and good semantics. Please note that the `section` tag should be used only as a main container. 😄 

- [ ] Kindly I suggest you follow the Figma template to make your design match the requirement, like margin, padding, font size, and font weight. Please go through this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=jtuuHJ5V6aki4hPg-0) to follow the Figma template.

> ### The Figma design
>
><img width="312" alt="Screenshot 2023-05-01 at 08 11 18" src="https://user-images.githubusercontent.com/91237525/235449890-2be3690c-ccdd-4c1c-b59a-d7173fb40ba3.png">
>
> ### Your design
>
><img width="377" alt="Screenshot 2023-05-01 at 07 58 01" src="https://user-images.githubusercontent.com/91237525/235449630-2160e217-bd68-4a1a-b2c6-00f4982a43c4.png">